packages/salesforcedx-vscode-core/test/jest/commands/deployExecutor.test.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { ContinueResponse } from '@salesforce/salesforcedx-utils-vscode';
+import { ComponentSet } from '@salesforce/source-deploy-retrieve';
+import * as fs from 'fs';
+import { DeployExecutor } from '../../../src/commands/DeployExecutor';
+import { WorkspaceContext } from '../../../src/context/workspaceContext';
+import { SourceTrackingService } from '../../../src/services';
+
+const dummyProjectPath = '/a/project/path';
```

Comment:
hum...I'm kinda surprised this doesn't complain that `dummyProjectPath` isn't defined during the mocking of `source-deploy-retrieve`.  jest.mock hoists to the verify beginning of execution so it doesn't have access to var declared outside of it.  You can move `dummyProjectPath` inside the handler function of the mock and just have another var in the test to expect against.

Reply:
I was able to remove this entirely since the test wasn't using it anymore.  I think that workspaceContext getInstance WI helped make the unit tests much simpler to implement.